CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# DeepSensor: Contributing Guidelines
+
+🎉🎈🍰 Welcome to DeepSensor repository! 🍰🎈🎉
+
+💫🐥 ☀️We're excited that you're here and want to contribute.☀️ 🐥💫
+
+Thank you for considering a contribution to the DeepSensor project! DeepSensor is a Python package dedicated to modeling environmental data with neural processes. With DeepSensor, we aim to provide a user-friendly, effective, and robust solution for the environmental research community. Our vision is <VISION>. By joining our efforts, you're helping push the boundaries of environmental science and machine learning, making strides towards a clearer understanding of our world.
```

Comment:
@tom-andersson We had been discussing a vision statement for DeepSensor. I can remove it for now, unless you have anything prepared or even drafted that we can slot in here?

Reply:
Here's something for now. Tried to keep it to a reasonable length. What do you think?

> Thank you for considering a contribution to DeepSensor! The DeepSensor Python package is motivated by successes in applying convolutional neural processes (ConvNPs) to a range of environmental prediction tasks: sensor placement, forecasting, downscaling, and satellite gap-filling. Behind the success of ConvNPs is the ability to efficiently fuse multi-modal and multi-resolution data, handle missing observations, and capture prediction uncertainty. DeepSensor aims to facilitate further research by plugging together the `xarray`, `pandas`, and `neuralprocesses` packages with a user-friendly interface that enables rapid experimentation. Now, our developers and users form an open-source research community whose vision is to accelerate the next generation of environmental ML research. We are eager to grow DeepSensor from a young Python package into a large and dedicated software community. By joining our efforts, you will be helping to push the frontiers of environmental sciences.